packages/evolution-frontend/src/components/pageParts/validations/InterviewListComponent.tsx

```diff
@@ -172,7 +174,14 @@ const InterviewListComponent: React.FunctionComponent<InterviewListComponentProp
                             title={props.t('admin:persons')}
                         />
                     </React.Fragment>
-                )
+                ),
+                label: props.t('admin:HouseholdSize'),
+                enableSortBy: true
+            },
+            {
+                accessor: 'responses.household.carNumber',
```

Comment:
Add a TODO that says this should be project specific, like on line 133.

Reply:
removed car number completely, not needed unless someone asks for it.